app/routes/lajidata.tsx

```diff
@@ -0,0 +1,40 @@
+import { json, LoaderFunction } from "@remix-run/node";
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const url = new URL(request.url);
+  const bounds = url.searchParams.get("bounds");
+  const accessToken = process.env.ACCESS_TOKEN;
+
+  if (!bounds || !accessToken) {
```

Comment:
Missing access token is server error (500) and not client error (400).

Reply:
Fixed

Reply:
Not quite, remove `!accessToken` part from here.